src/styles/s3.css

```diff
@@ -0,0 +1,101 @@
+
+/**
+ * @author Jamie jamie.h@inwinstack.com
+ */
+ 
+/* border radius ----------------------------------------- */
+.radius-4px {
+    border-radius: 4px;
+}
+
+.radius-6px {
+    border-radius: 6px;
+}
+
+.radius-10px {
+    border-radius: 10px;
+}
+
+/* panel ------------------------------------------------- */
+.panel-signup,
+.panel-login {
+    border-radius: 4px;
+    left: 0;
+    margin: 0 auto;
+    position: absolute;
+    right: 0;
+    top: 200px;
+    width: 360px;
+}
+
+.panel-title {
+    background: #616161;
+    color: #fff;
+    height: 50px;
+    padding-top: 12px;
+}
+
+.panel-body {
+    padding: 0 8px 8px 8px;;
+}
+
+.panel-footer {
+    margin: 20px;
+}
+
+/* button ------------------------------------------------ */
+.btn-block {
+    display: block;
+    margin: 6px 0;
+    width: 100%;
+}
+
+button md-progress-circular {
+    position: absolute;
+    top: 3px;
+    margin: 0 auto;
+    left: 0;
+    right: 0;
+}
+
+/* icon ---------------------------------------------------*/
+.icon-gray {
```

Comment:
The class name should not be named as property value. It should be meaningful, ex. icon-disabled or icon-default